### PR TITLE
Feat/has key

### PR DESCRIPTION
**Pull request by NgrillSolutions**

### CODE REVIEW

File: src/localization.tsx [hunk 1]
```diff
@@ -53,6 +53,9 @@ type SetLanguageFunc = (language: string | null) => any;
  */
 type TFunc<T> = (key: Leaves<T>, context?: Record<string, any>) => string;
 
+type THasKeyFunc<T> = (key: Leaves<T>) => boolean;
```
Comment (peterkarrer) [inline src/localization.tsx:56]: Here we should allow to pass any `string` instead of `Leaves<T>`

**Review by peterkarrer — CHANGES_REQUESTED** [review]
Besides the one small type change, this looks great!

File: src/localization.tsx (hunk 1 above)
Comment (NgrillSolutions) [reply src/localization.tsx]: makes sense. I changed that in the next commit.

**Review by peterkarrer — APPROVED** [review]

**Comment by peterkarrer** [comment]
This has been released in `1.0.0-alpha.59`.

**Pull request merged**